app.py

```diff
@@ -52,7 +52,60 @@ def mirror(name):
     return create_response(data)
 
 
-# TODO: Implement the rest of the API here!
+@app.route("/users")
+def get_users():
```

Comment:
General comment: please add type hints https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html
(it's still optional in python, but we do try to use it in our code)